src/state/swap/hooks.ts

```diff
@@ -256,7 +256,11 @@ function parseCurrencyFromURLParameter(urlParam: any, chainId: ChainId): string
   if (typeof urlParam === 'string') {
     const valid = isAddress(urlParam)
     if (valid) return valid
-    return nativeOnChain(chainId).symbol as string
+    const symbolNative = nativeOnChain(chainId).symbol as string
+    /**
+     * handle case ex: at chainId=1 (ethereum) urlParam=BNB => we will return '' to redirect /swap
```

Comment:
Cant understand context here. Can you make it more clear?

Reply:
`case ex: at chainId=1 (ethereum) urlParam=BNB` what does these mean?

Reply:
you are on ethereum. your url   `/swap?networkId=1&inputCurrency=ETH&outputCurrency=...`
you switch network to bnb:  your network changed, params still inputCurrency=ETH, input currently select BNB. that is confused.    
I fixed it by return '' => our web will redirect to /swap (remove `inputCurrency=ETH....`)

Reply:
Why do we change network but still keep `inputCurrency` and `outputCurrency` on query? We should remove them so the page gonna choose default pair, shouldn't we?

It mean `/swap?networkId=1&inputCurrency=ETH&outputCurrency=...` => change network BNB => `/swap?networkId=56`

Reply:
it's ok. I'm considering

Reply:
updated. continue review bro. 
QC tested again.